src/amchart/CommonSerial.js

```diff
@@ -489,12 +489,12 @@
 
     CommonSerial.prototype.amFormatColumns = function(ColArr) {
         var context = this;
-        this._categoryField = this._columns[0];
-        this._chart.categoryField = this._columns[0];
+        this._categoryField = this.columns()[0];
+        this._chart.categoryField = this.columns()[0];
         this._valueField = [];
         this._openField = [];
         this._closeField = [];
-        this._columns.slice(1, this._columns.length).forEach(function(col, idx) {
+        this.columns().slice(1, this.columns().length).forEach(function(col, idx) {
             context._valueField.push(col);
         });
```

Comment:
This could have been simplified in the same way as CommonRadar:
**this._valueField = this.columns().filter(function (d, i) { return i > 0; });**